### DISCUSSION TITLE
How to listen for dnstap source from network

### DISCUSSION THREAD

**Discussion by peterand-pa** [vectordotdev/vector#19829] [Q&A]
Source type dnstap seems to enforce socket_path .
I've tried putting 0.0.0.0:6053 as the path ,  but all that did was created a socket file on disk.
Can someone advise ?

**Comment by jszwedko** [comment]
The dnstap source only supports unix sockets at the moment. See https://github.com/vectordotdev/vector/issues/19735 and https://github.com/vectordotdev/vector/issues/15203.

**Comment by peterand-pa** [comment]
This was my other alternative, but I get this error:
Failed deserializing frame. error=Error parsing protobuf: DecodeError { description: "invalid tag value: 0", stack: [] }

> sources:
>    dnstap:
>     type: socket
>     address: 0.0.0.0:6053
>     mode: tcp
>     decoding:
>       codec: protobuf
>       protobuf:
>         message_type:  dnstap.Dnstap
>         desc_file: /etc/vector/dnstap.desc
>     host_key: host
> 
>
